frontend/packages/console-app/console-extensions.json

```diff
@@ -304,5 +304,19 @@
         "defaultValue": false
       }
     }
+  },
+  {
+    "type": "console.user-preference/item",
+    "properties": {
+      "id": "console.theme",
+      "label": "%console-app~Theme%",
+      "groupId": "general",
+      "description": "%console-app~Console will appear as per the selected theme.%",
+      "field": {
+        "type": "custom",
+        "component": { "$codeRef": "userPreferences.ThemeField" }
+      },
```

Comment:
@vikram-raj Valid point here, if you prepare a version with `useUserSettings(..., sync=true)` and the sync feature doesn't work for any reason. Please let me know so that we can check/debug together.